core/src/test/java/org/infinispan/xsite/XSiteCacheConfigurationTest.java

```diff
@@ -73,18 +76,20 @@ public void testSameBackupDefinedMultipleTimes() {
       cb.build();
    }
 
-   @Test(expectedExceptions = CacheConfigurationException.class)
+   @Test(expectedExceptions = CacheConfigurationException.class, expectedExceptionsMessageRegExp = "ISPN\\d+: The 'site' must be specified!")
```

Comment:
Could this be refactored so instead using the @ expected exceptions we use the Exceptions class ? while I'm writing this comment I'm not sure anymore if we want to use the expectedExceptions in the annotation or the org.infinispan.commons.test.Exceptions class. 
Could you please tell us which is the best way or both ways are equally valid now @danberindei ?

Reply:
I can change to whatever version you want :)